### PR TITLE
Update info text and button once more.

### PR DESCRIPTION
**Pull request by mariaschuld**
As we discussed in the meeting, we should be clearer on what the user can expect from registering. I hope this is better?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -106,10 +106,13 @@ <h3>Machine Learning and Artificial Intelligence Conference</h3>
       <br>
 
       <div class="row intro-text">
-        <p>This year's IndabaX South Africa will be held as a Roadshow of various
-          fun events, <br>
-          taking place from <b>3 September to 26 November</b>. <br>
-          Scroll down to find out more, or join the mailing list to stay in the loop! </p>
+        <p>"This year's IndabaX South Africa will be held as a Roadshow of events and competitions <br>
```
Comment (mariaschuld) [inline index.html:109]:
```suggestion
        <p>This year's IndabaX South Africa will be held as a Roadshow of events and competitions <br>
```

File: index.html [hunk 2]
```diff
@@ -106,10 +106,13 @@ <h3>Machine Learning and Artificial Intelligence Conference</h3>
...
-          fun events, <br>
-          taking place from <b>3 September to 26 November</b>. <br>
-          Scroll down to find out more, or join the mailing list to stay in the loop! </p>
+        <p>This year's IndabaX South Africa will be held as a Roadshow of events and competitions <br>
+          hosted by South Africa's top ML labs, and taking place from 
+          <b>3 September to 26 November</b>.<br>
+          Register to receive all important updates from us and to get your link to join the Discord community forum <br>
+          where you can meet people, share knowledge and ask experts for help. <br>
+        Alternatively, you can check out the schedule below to sign up for individual events.
+        </p>
```
Comment (mariaschuld) [inline index.html:115]: I'm not sure if this is too long...?

File: index.html [hunk 3]
```diff
@@ -118,7 +121,7 @@ <h3>Machine Learning and Artificial Intelligence Conference</h3>
         <div class="col-md-4 col-xs-12">
           <h3>
             <a class="btn btn-success btn-block" href="http://eepurl.com/hCE_C5">
-              Join our community
+              Register
```
Comment (mariaschuld) [inline index.html:124]: But then this one is clearly explained above!

File: index.html (hunk 2 above)
Comment (Sicelukwanda) [reply index.html:115]: seems fine

**Pull request merged**